### PR TITLE
Refactor registry related envs

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- make docker registry secret more readable
- remove unneeded variables
- serverAddress is no longer needed bacause: external registry didn't use it and with this change the internal also.

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/17459

### CODE REVIEW

File: resources/serverless/values.yaml [hunk 1]
```diff
@@ -277,9 +277,8 @@ dockerRegistry:
   gatewayCert: "kyma-gateway-certs"
   username: "{{ randAlphaNum 20 | b64enc }}" # for gcr "_json_key"
   password: "{{ randAlphaNum 40 | b64enc }}" # for gcr data from json key
-  internalServerAddress: '{{ template "registry-fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.global.registryServicePort }}'
-  registryAddress: '{{ template "registry-fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.global.registryServicePort }}'
-  serverAddress: "localhost:{{ .Values.global.registryNodePort }}"
+  registryAddress: ""
+  serverAddress: ""
```
Comment (dbadura) [inline resources/serverless/values.yaml:281]: I can describe the difference for those fields

**Review by pPrecel — APPROVED** [review]

**Pull request merged**